src/main/java/io/strimzi/kafka/bridge/BridgeContentType.java

```diff
@@ -16,9 +16,16 @@ public class BridgeContentType {
     /** JSON encoding with BINARY embedded format */
     public static final String KAFKA_JSON_BINARY = "application/vnd.kafka.binary.v2+json";
 
+
+    /** JSON encoding with Plain text embedded format */
+    public static final String KAFKA_JSON_TEXT = "application/vnd.kafka.string.v2+json";
```

Comment:
I think we agreed on using `text` instead of `string` on the issue.

Reply:
Aren't they the same thing? I couldn't find any reference for `text`.

Reply:
We were discussing here to use `text` instead of `string` https://github.com/strimzi/strimzi-kafka-bridge/issues/739#issuecomment-1727337478 so it should be `application/vnd.kafka.text.v2+json`

Reply:
Yes, you are right. 
I just didn't know that `application/vnd.kafka.text.v2+json` works, as I didn't find any reading material online.

Reply:
`application/vnd.kafka.text.v2+json` this is not something you could find online.
This is pretty custom like the others already in place and it is not a well defined standards in the HTTP land.

Reply:
> application/vnd.kafka.text.v2+json

It seems impossible, but I did not find anything on the internet tbh. That is why I was thinking it's an invalid content type. Anyway, now I am sure it is valid.

Reply:
HTTP Content-Type (bringing the Media type) is something that you can define for your own application. There are some standard ones like `text/plain`, `application/json` and so on but it doesn't mean that you can't create your own.
The Strimzi bridge has its own content type. You can understand that because of the `vnd.` prefix which means to be vendor specific (this case Strimzi specific) and it defines a "Subtype".
Also the `+json` suffix specifies the structure of the media type to be always JSON.
So you will never find information about `application/vnd.kafka.text.v2+json` out of Strimzi project.
More details here: https://en.wikipedia.org/wiki/Media_type